packages/core/src/modules/proofs/formats/dif-presentation-exchange/DifPresentationExchangeProofFormatService.ts

```diff
@@ -218,6 +221,24 @@ export class PresentationExchangeProofFormatService implements ProofFormatServic
     return { attachment, format }
   }
 
+  private verifyUsingAnoncreds2023(
+    presentation: W3cVerifiablePresentation,
+    presentationSubmission: PresentationSubmission
+  ) {
+    if (presentation.claimFormat !== ClaimFormat.LdpVp) return false
+
+    const descriptorMap = presentationSubmission.descriptor_map
+
+    const verifyUsingDataIntegrity = descriptorMap.every((descriptor) => descriptor.format === ClaimFormat.DiVp)
+    if (!verifyUsingDataIntegrity) return false
+
+    if (Array.isArray(presentation.proof)) {
+      return presentation.proof.some((proof) => proof.cryptosuite?.includes('anoncreds-2023'))
```

Comment:
Maybe we can make `anoncreds-2023` a const that we use everyweyre as it's now harcoded in quite some places